.eslintrc

```diff
@@ -52,8 +52,7 @@
       {
         "alphabetize": {
           "order": "asc"
-        },
-        "groups": ["builtin", "external", "sibling", "parent", "index"]
+        }
```

Comment:
We're now getting the default groups of `["builtin", "external", "parent", "sibling", "index"]`.

Basically the same, except parent comes before sibling.

I'd like to actually do away with "groups" and just alphabetically sort the import paths. But at this point it makes sense to go with what we landed on for Traject and the shared configs.